### PR TITLE
Added Branching Strategy in README.md

### PR DESCRIPTION
**Pull request by ruchita-nathani**
Added branching strategy  in readme.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,12 @@
...
+## Branching Strategy  
+
+**main**: It contains fully stable production code
+* **dev**: It contains stable under-development code
+    
+  * **epic**: It contains a module branch. Like high level of feature. For example, we have an authentication module then we can create a branch like "epic/authentication"
+
+    * **feature**: It contains specific features under the module. For example, under authentication, we have a feature called registration. Sample branch name: "feature/registration"
+
+    * **bugfix**: [It contains bug fixing during the testing phase and  branch name start with the issue number for example "bugfix/3-validate-for-wrong-user-name"
```
Comment (felix-zailskas) [inline README.md:12]: Here is an extra [ in this line.

**Review by felix-zailskas — CHANGES_REQUESTED** [review]
Adjust the mentioned line, please. Otherwise good addition.

File: README.md (hunk 1 above)
Comment (ruchita-nathani) [reply README.md]: Issue Resolved. Please check now.

**Review by felix-zailskas — APPROVED** [review]

**Comment by Tims777** [comment]
I just merged your PR, which is therefore our first PR ever in this project. Good job :)

**Pull request merged**